modules/benchmark/R/load_data.R

```diff
@@ -34,13 +34,27 @@ load_data <- function(data.path, format, start_year = NA, end_year = NA, site =
     fcn <- match.fun(fcn1)
   } else if (exists(fcn2)) {
     fcn <- match.fun(fcn2)
+  } else if (!exists(fcn1) & !exists(fcn2) & require(bd)) { 
+    #To Do: call to DAP to see if conversion to csv is possible
+    #Brown Dog API call through BDFiddle, requires username and password
+    key   <- get_key("https://bd-api.ncsa.illinois.edu",username,password)
+    token <- get_token("https://bd-api.ncsa.illinois.edu",key)
+    #output_path = where are we putting converted file?
+    converted.data.path <- convert_file(url = "https://bd-api.ncsa.illinois.edu", input_filename = data.path, 
+                                        output = "csv", output_path = output_path, token = token)
+    if (is.na(converted.data.path)){
+      PEcAn.utils::logger.error("Converted file was not returned from Brown Dog")
+    }
+    #not doing anything about mimetypes not convertible by BD right now
+    fcn <- match.fun(fcn2)
```

Comment:
Sorry for not catching this earlier, but just realized that after the conversion you set fcn to fcn2, which is the ORIGINAL mime-type function. But if we got here we know that we can't load that mime-type and so instead we converted to a different mime-type. So we need to load using that different mime-type. Specifically, this code should probably be
```
fcn <- match.fun("load_csv")
```